### PR TITLE
Docs: support bundle section for 9.5 What's New

### PR DESCRIPTION
**Pull request by IevaVasiljeva**

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v9-5.md
Comment (IevaVasiljeva) [inline docs/sources/whatsnew/whats-new-in-v9-5.md:92]: @eleijonmarck could you please double-check these changes? The linter was complaining about spelling, so I read through this section and I think you can simplify the last paragraph a bit.

Comment (eleijonmarck) [reply docs/sources/whatsnew/whats-new-in-v9-5.md:92]: yea i like the shortening of the paragraph. thx for looking into it

**Review by eleijonmarck — APPROVED** [review]
lgtm

File: docs/sources/whatsnew/whats-new-in-v9-5.md
Comment (chri2547) [inline docs/sources/whatsnew/whats-new-in-v9-5.md:88]: Maybe: "This release comes with a feature toggle called `onlyExternalOrgRoleSync` that you can enable. Once enabled, users signing in to Grafana cannot change organization roles synchronized with an external authentication provider."

File: docs/sources/whatsnew/whats-new-in-v9-5.md
Comment (chri2547) [inline docs/sources/whatsnew/whats-new-in-v9-5.md:90]: And then remove this first sentence.

File: docs/sources/whatsnew/whats-new-in-v9-5.md
Comment (chri2547) [inline docs/sources/whatsnew/whats-new-in-v9-5.md:92]:
```suggestion
You can also prevent the synchronization of organization roles from the authentication provider. For more information, refer to [skip org role sync](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync).
```

File: docs/sources/whatsnew/whats-new-in-v9-5.md
Comment (chri2547) [inline docs/sources/whatsnew/whats-new-in-v9-5.md:98]: How is creating a support bundle useful to the user? Do they do something with it, like send it to us so that we can troubleshoot?

Comment (chri2547) [inline docs/sources/whatsnew/whats-new-in-v9-5.md:98]:
```suggestion
Support bundles provide a simple way to collect information about your Grafana instance through Grafana's user interface. In a few clicks, you can create a support bundle containing data about migrations, plugins, settings, and more. This is particularly useful when troubleshooting issues with your Grafana instance.
```

**Review by chri2547 — APPROVED** [review]
Comments and suggestions. I am approving so as not to block this PR.

**Pull request merged**
